app/uk/gov/hmrc/economiccrimelevyregistration/views/CheckYourAnswersView.scala.html

```diff
@@ -14,16 +14,25 @@
  * limitations under the License.
  *@
 
+@import uk.gov.hmrc.economiccrimelevyregistration.models.RegistrationType._
+
 @this(
         layout: templates.Layout,
         formHelper: FormWithCSRF,
         govukSummaryList: GovukSummaryList,
         govukButton: GovukButton
 )
 
-@(eclDetails: SummaryList, organisationDetails: SummaryList, contactDetails: SummaryList, isAmendment: Boolean)(implicit request: Request[_], messages: Messages)
+@(eclDetails: SummaryList,
+        organisationDetails: SummaryList,
+        contactDetails: SummaryList,
+        isAmendment: Boolean,
+        eclReference: Option[String])(implicit request: Request[_], messages: Messages)
 
-@layout(pageTitle = title(messages("checkYourAnswers.title"))) {
+@layout(pageTitle = title(messages("checkYourAnswers.title")),
+        registrationTypeForTopBanner = if(isAmendment) Some(Amendment) else Some(Initial),
```

Comment:
why do we need the isAmendment boolean here? Can't we just pass the registrationType?